esp/platform/espp.hpp

```diff
@@ -20,13 +20,21 @@
 
 #include "espthread.hpp"
 #include "espcfg.ipp"
+#include "jlog.ipp"
 
 typedef ISocket * isockp;
 
 
 typedef MapBetween<int, int, isockp, isockp> SocketPortMap;
 typedef CopyReferenceArrayOf<ISocket> SocketPortArray;
 
+class CEspElevatedLoggingFilter : public CategoryLogMsgFilter
+{
+public:
+    CEspElevatedLoggingFilter() : CategoryLogMsgFilter(MSGAUD_all, MSGCLS_elevated, DebugMsgThreshold, false) {}
+    bool mayIncludeCategory(const LogMsgCategory & cat) const { return (cat.queryClass() & classMask); }
```

Comment:
Why does the check not consider the message level? Values > DebMsgThreshold should be excluded.

Reply:
why would we want to exclude any message based on threshold? would that follow existing behavior?
DebMsgThreshold  is 80, DBGLOG() in baremetal I believe logs all entries.

Reply:
Audience isn't checked because the filter accepts any. Class is checked for reasons obvious to me. I'm unsure why level isn't checked when `cat` may specify a higher value.

I am admittedly unfamiliar with filters, and maybe this does make sense. I'm really just asking for an explanation so I can understand it.